user_docs/en/userGuide.t2t

```diff
@@ -1590,6 +1592,15 @@ This can be useful if it is hard to distinguish between pronunciation of symbols
 
 The delayed character description will be cancelled if other text is spoken during that time, or if you press the ``control`` key.
 
+====Switch between the following speech modes: ====[SpeechModesDisabling]
+This checkable list allows to select which speech modes are included when cycling between them using ``NVDA+s``.
+Modes which are unchecked are excluded.
+By default all modes are included, also if any new mode is added to NVDA it will also be included by default.
+
+For example if you do not need to use "beeps" mode and the "on-demand" mode you should uncheck these two, and keep both "off" and "talk" checked.
+Note that it is necessary to check at least two modes, and one of "talk" or "on-demand" has to be checked, so that you are not left without any speech.
+This is validated when pressing Ok or Apply, and an appropriate error message is shown when these constraints are not meet.

```

Comment:
IMO this sentence should just be removed.
1. The fact that pressing OK or Apply validates the dialog is not at all specific to this option.
2. There is no need to mention error dialogs in the User Guide; you have already written that at least two modes should be selected.

Reply:
I've mentioned the error messages for consistency with the description of the validation for NVDA modifier keys. Should it be removed also from their description in your opinion? Note that pressing Okay or Apply does no validation in other dialogs really.

Reply:
I agree that, while at it, the paragraph dedicated to NVDA key should be reworded. It is even wrong today, since it suggests implicitly that the settings dialog remains open after you have pressed OK in the error dialog.

A positive sentence such as:
"You have to check at least one key, so that you can still access NVDA commands."

The following sentences should be removed:
> Therefore, The NVDA settings dialog will display an error message if all keys are unselected when pressing Ok or Apply.
> After dismissing the error message, you must select at least one before being able to press Ok to dismiss the dialog properly.

Reply:
The paragraph in use guide was reworded in 8bdf346 Note that in this commit settings dialog indeed remains open until user checks at least one modifier.